### DISCUSSION TITLE
Formatting SQL queries inside $queryRaw

### DISCUSSION THREAD

**Discussion by leo-petrucci** [prisma/prisma#5312] [Q&A]
My app requires a couple of complex SQL queries I can't create with prisma, so I need to use `queryRaw`. VsCode has a bunch of extentions that recognise `sql` template literals and format them accordingly, but is there something similar for `queryRaw`?

My SQL formatting is all over the place

**Comment by nikolasburk** [comment]
Hey Leo (nice to see you around in our community so much 😄 )

I've raised this internally and am still awaiting a reply but my hunch is that this is currently not possible. But I think it's certainly a valid use case and might even be smth that we want to add to the Prisma VS Code extension if possible. To bring this to the attention of our Product and Engineering teams, I'd probably recommend you to [create a feature request in the `language-tools` repo](https://github.com/prisma/language-tools/issues/new/choose).

**Reply by nikolasburk** [reply]
Quick update: Just got the confirmation from our engineers that this is not possible, but it's definitely a great idea so if you could go ahead and create the FR that would be super helpful 🙏

**Reply by leo-petrucci** [reply]
No worries! Wanted to check if I was just a bad googler before opening up a FR 🙂
I'll create a Feature Request asap, thank you for the response

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
